### ISSUE TITLE
Implement remaining barricades

### ISSUE THREAD

**Issue by paroxum** [Facepunch/rust#115]
The concrete and sandbags barriers should be straightforward to implement I think. 

While the stone barricade is only a 1x1m block so we have to figure out if we let players place it one by one or if we already put 3 side by side in our prefab to stick on our 3m grid.

**Comment by OverlordJ19** [comment]
No, they should be 1/1. Also, none of the barricades should have to follow building block procedure, just like wood barricades outside, or a furnace inside, they should be a very custom placeable item.
